vac/raw/eth-secpm.md

```diff
@@ -824,6 +824,33 @@ These credentials MUST use the digital signature key pair associated to the Ethe
 - A Delivery Service storage system MUST verify KeyPackages before storing them.
 - Interaction diagrams involving the DS do not change.
 
+## Consideration related to the onchain component of the protocol
+- Each group is managed by a separate smart contract.
+
+- Below follows a description of Alice adding Bob as a member of a new group.
+    1. Alice creates the smart contract associated to the group. This smart contract MUST include an ACL.
+    2. Alice adds Bob’s Ethereum address to the ACL. 
```

Comment:
On this step I think we also need to add Bob public key which we will use for verification